### PR TITLE
sign the intermediate signer

### PR DESCRIPTION
**Pull request by deads2k**
The cert rotation on a signer didn't properly sign the intermediate.  This updates it to do that.

/assign @sjenning

### CODE REVIEW

File: vendor/bitbucket.org/ww/goautoneg/.hg_archival.txt [hunk 1]
```diff
@@ -1,6 +0,0 @@
-repo: 848b351341922ce39becda978778724d5b58dbca
-node: 75cd24fc2f2c2a2088577d12123ddee5f54e0675
-branch: default
-latesttag: null
-latesttagdistance: 5
-changessincelatesttag: 5
```
Comment (eparis) [inline vendor/bitbucket.org/ww/goautoneg/.hg_archival.txt]: what's going on here?  do you have an old dep? or did the person who created this have an old dep?

Comment (deads2k) [reply vendor/bitbucket.org/ww/goautoneg/.hg_archival.txt]: > what's going on here? do you have an old dep? or did the person who created this have an old dep?

no idea.

**Comment by abhinavdahiya** [comment]
Can you point or include what this is getting, in terms of PR to library-go or something?

**Comment by derekwaynecarr** [comment]
@abhinavdahiya https://github.com/openshift/library-go/pull/228

**Comment by deads2k** [comment]
OLM

/retest

**Comment by deads2k** [comment]
@mfojtik @sttts I'm going to move that last commit into library-go so that we properly go failing and probably base availablility on it

File: vendor/github.com/openshift/library-go/pkg/operator/staticpod/controller/staticpodstate/staticpodstate_controller.go [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+func NewStaticPodStateController(
+	targetNamespace, staticPodName string,
+	kubeInformersForTargetNamespace informers.SharedInformerFactory,
+	operatorConfigClient v1helpers.StaticPodOperatorClient,
+	kubeClient kubernetes.Interface,
+	eventRecorder events.Recorder,
+) *StaticPodStateController {
+	c := &StaticPodStateController{
+		targetNamespace: targetNamespace,
+		staticPodName:   staticPodName,
```
Comment (mfojtik) [inline vendor/github.com/openshift/library-go/pkg/operator/staticpod/controller/staticpodstate/staticpodstate_controller.go]: @deads2k not clear to me how you get the current revision of the pod

**Comment by deads2k** [comment]
`F0207 23:23:30.990038       1 controllermanager.go:217] error starting controllers: failed to start certificate controller: error parsing CA cert file "/etc/kubernetes/static-pod-resources/secrets/csr-signer/tls.crt": {"code":1003,"message":"the PEM file should contain only one object"}`

**Comment by deads2k** [comment]
module.vpc.aws_route.to_nat_gw[3]: 1 error occurred

/tetest

**Comment by deads2k** [comment]
tagging so I get retests that will contain my configmaps https://github.com/openshift/release/pull/2842

File: vendor/bitbucket.org/ww/goautoneg/.hg_archival.txt (hunk 1 above)
Comment (tnozicka) [reply vendor/bitbucket.org/ww/goautoneg/.hg_archival.txt]: glide does bad stuff - it doesn't store `.git` but it does store the internal metadata for mercurial. given different mercurial store different metadata version, diffing gets impossible - we are fixing this post `glide update` in 
https://github.com/openshift/library-go/blob/b25329afa1ae9d7dfcd7c3d8d19940d7bb87afdd/alpha-build-machinery/make/targets/openshift/deps.mk#L16

**Comment by deads2k** [comment]
this is weird.  It works locally.  I wonder if ran the test before the payload was ready...

**Comment by deads2k** [comment]
/hold

this now has an illegal debug patch while I wait for a real upstream

**Comment by deads2k** [comment]
/hold cancel

found the lister bug.  Need to fix up the cert rotation listers now that we have the combined stuff

**Comment by deads2k** [comment]
fixed the tls.cert/tls.crt typo

**Comment by deads2k** [comment]
aws

/retest

**Comment by mfojtik** [comment]
metrics flake

/retest

**Comment by deads2k** [comment]
nodes are up.  The failures are the known apiserver flake

**Pull request merged**